Library/String+SimpleHTML.swift

```diff
@@ -17,20 +17,20 @@ public extension String {
    as html.
    */
   public func simpleHtmlAttributedString(
-    base base: Attributes,
+    base: Attributes,
          bold optionalBold: Attributes? = nil,
               italic optionalItalic: Attributes? = nil) -> NSAttributedString? {
-    let baseFont = (base[NSFontAttributeName] as? UIFont) ?? UIFont.systemFontOfSize(12.0)
+    let baseFont = (base[NSFontAttributeName] as? UIFont) ?? UIFont.systemFont(ofSize: 12.0)
 
     // If bold or italic are not specified we can derive them from `font`.
     let bold = optionalBold ?? [NSFontAttributeName: baseFont.bolded]
     let italic = optionalItalic ?? [NSFontAttributeName: baseFont.italicized]
 
-    guard let data = self.dataUsingEncoding(NSUTF8StringEncoding) else { return nil }
+    guard let data = self.data(using: String.Encoding.utf8) else { return nil }
 
-    let options: [String:AnyObject] = [
+    let options: [String:Any] = [
       NSDocumentTypeDocumentAttribute: NSHTMLTextDocumentType,
-      NSCharacterEncodingDocumentAttribute: NSUTF8StringEncoding
+      NSCharacterEncodingDocumentAttribute: String.Encoding.utf8.rawValue
```

Comment:
weirdly without this change the compiler didnt complain but there was a runtime crash :/